app/components/updatetypebanner/updatetypebanner.js

```diff
@@ -0,0 +1,84 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { translate } from 'react-i18next';
+import { push } from 'connected-react-router';
+import { connect } from 'react-redux';
+
+export const UpdateTypeBanner = translate()((props) => {
+  const {
+    onClick,
+    onClose,
+    patient,
+    trackMetric,
+    t,
+    push,
+  } = props;
+
+  const getMessageText = () => {
+    return t('Complete your profile.');
+  };
+
+  const getButtonText = () => {
+    return t('Update My Profile');
+  };
+
+  const getLearnMoreText = () => {
+    return t('Add your birthday, diagnosis date, and type');
+  };
```

Comment:
Makes sense -- just followed the pattern from the other banners, I figured it would make it more straight forward when we have time to refactor them... when there are more of them, or whenever that is possible